sql/semisync_master.cc

```diff
@@ -641,6 +611,7 @@ int Repl_semi_sync_master::report_reply_packet(uint32 server_id,
 
   rpl_semi_sync_master_get_ack++;
   report_reply_binlog(server_id, log_file_name, log_file_pos);
+  DEBUG_SYNC(current_thd, "semisync_got_reply");
```

Comment:
@bnestere can you help me with Debug Sync? The program appears never to hit this sync point.

Good call on that there’s only one singular ACK receiver thread, by the way.